crates/polars-core/src/frame/group_by/aggregations/dispatch.rs

```diff
@@ -109,19 +111,29 @@ impl Series {
         use DataType::*;
 
         match self.dtype() {
+            Boolean => self.cast(&Float64).unwrap().agg_mean(groups),
```

Comment:
should this be `agg_median`?

can you also add a test case for this please, asserting that for both
```python
In [12]: pl.Series([True, True, False]).median()
Out[12]: 1.0

In [13]: pl.Series([True, True, False]).to_frame('a').group_by(1).agg(pl.col('a').median())
Out[13]: 
shape: (1, 2)
┌─────────┬──────────┐
│ literal ┆ a        │
│ ---     ┆ ---      │
│ i32     ┆ f64      │
╞═════════╪══════════╡
│ 1       ┆ 0.666667 │
└─────────┴──────────┘
```
the output should be `1.0`?

Reply:
Yes, good catch, thank you.